packages/dds/merge-tree/src/attributionPolicy.ts

```diff
@@ -190,12 +190,17 @@ function createPropertyTrackingMergeTreeCallbacks(
 
 function combineMergeTreeCallbacks(callbacks: AttributionCallbacks[]): AttributionCallbacks {
 	return {
-		delta: (...args) => callbacks.forEach(({ delta }) => delta(...args)),
-		maintenance: (...args) => callbacks.forEach(({ maintenance }) => maintenance(...args)),
+		delta: (...args): void => {
+			for (const { delta } of callbacks) delta(...args);
+		},
+		maintenance: (...args): void => {
+			for (const { maintenance } of callbacks) maintenance(...args);
+		},
 	};
 }
 
 /**
+ * Creates an {@link AttributionPolicy} for inserted content only.
  * @returns An {@link AttributionPolicy} which tracks only insertion of content.
```

Comment:
Nit: in the cases where a `@returns` block more or less just echos what the summary block says, I think it's preferrable to remove the `@returns` block. Ideally, specialized blocks should only be used if they offer additional information.